tests/test_factory_fields.py

```diff
@@ -74,3 +76,25 @@ class MyFactory(ModelFactory):
         name = Ignore()
 
     assert MyFactory.build().name is None
+
+
+def test_post_generation():
+
+    random_delta = timedelta(days=random.randint(0, 12), seconds=random.randint(13, 13000))
+
+    def add_timedelta(name, values, **kwds):
+        assert name == "to_dt"
+        assert "from_dt" in values
+        assert isinstance(values["from_dt"], datetime)
+        return values["from_dt"] + random_delta
+
+    class MyModel(BaseModel):
+        from_dt: datetime
+        to_dt: datetime
+
+    class MyFactory(ModelFactory):
+        __model__ = MyModel
+        to_dt = PostGenerated(add_timedelta)
```

Comment:
please add a test with multiple types of fields, including multiple `PostGenerated` - we need some sanity checks.